Consul/Operator.cs

```diff
@@ -442,6 +442,16 @@ public Task<QueryResult<AutopilotState>> AutopilotGetState(QueryOptions q, Cance
         {
             return _client.Get<AutopilotState>("/v1/operator/autopilot/state", q).Execute(cancellationToken);
         }
+
+        public Task<QueryResult<OperatorUsageState>> OperatorUsageGetState(CancellationToken cancellationToken = default)
```

Comment:
I think the method name is slightly too long and not accurate. Perhaps, `OperatorGetUsage` or `OperatorUsageInformation` ?